backend/config/postgres-integration.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/go-pg/pg/v10"
+)
+
+// function to connect to Postgres
+func Connect2db() *pg.DB {
+	fmt.Println("Connection Initiated")
+	opts := &pg.Options{
+		Network:               "",
+		Addr:                  "localhost:5432",
+		User:                  "postgres",
+		Password:              "gitops",
+		Database:              "managed-gitops-postgres",
+		ApplicationName:       "",
```

Comment:
My guess is that we can remove these lines:
```go
		Network:               "",
```
and
```go
		ApplicationName:       "",
		DialTimeout:           0,
		ReadTimeout:           0,
		WriteTimeout:          0,
		MaxRetries:            0,
		RetryStatementTimeout: false,
		MinRetryBackoff:       0,
		MaxRetryBackoff:       0,
		PoolSize:              0,
		MinIdleConns:          0,
		MaxConnAge:            0,
		PoolTimeout:           0,
		IdleTimeout:           0,
		IdleCheckFrequency:    0,
```

Reply:
Yes, it would be clean to have this removed if it was not required! :hugs: 
Done, done!